content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.getActualMinimum()'
+Description: 'Returns the minimum value that the specified calendar field could have based on the time value of the calendar.'
```

Comment:
This description uses the same wording as the definitions on other sites-
https://www.tutorialspoint.com/java/util/calendar_getactualminimum.htm
https://www.geeksforgeeks.org/calendar-getactualminimum-method-in-java-with-examples/

There is no plagiarism permitted in docs, all content must be original. Please update the description using your own words. Also update line 16 to be in alignment with the new description.